lib/util/customError.js

```diff
@@ -10,7 +9,9 @@ var CustomError = function(name, message) {
   this.message = message;
 };
 
-
+/**
+ * Creates custom errors to primarily help keep the program from crashing.
```

Comment:
That depends entirely on the usage of this module. Just say it's a module for creating custom errors. You don't know how this will be used.